sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/TPCQueryBenchmarkArguments.scala

```diff
@@ -20,14 +20,11 @@ package org.apache.spark.sql.execution.benchmark
 import java.util.Locale
 
 
-class TPCDSQueryBenchmarkArguments(val args: Array[String]) {
-  var dataLocation: String = sys.env.getOrElse("SPARK_TPCDS_DATA", null)
+class TPCQueryBenchmarkArguments(val args: Array[String], val dataLocationEnv: String) {
+  var dataLocation: String = sys.env.getOrElse(dataLocationEnv, null)
   var queryFilter: Set[String] = Set.empty
   var cboEnabled: Boolean = false
 
-  parseArgs(args.toList)
```

Comment:
why remove the invoke of `parseArgs` and `validateArguments`, these 2 functions will be unused after this pr

Reply:
My bad, it's fixed now